ProcessesApi/ExceptionMiddlewareExtensions.cs

```diff
@@ -10,7 +10,7 @@
 using System.Linq;
 using System.Threading.Tasks;
 
-namespace BaseApi
+namespace ProcessesApi
```

Comment:
This file can be deleted and you can use to the Middleware nuget package instead